pkg/remote/builtin/github/github.go

```diff
@@ -34,12 +37,14 @@ type GitHub struct {
 
 func New(conf *config.Config) *GitHub {
```

Comment:
Sorry, we of course wait RFC fixed :sweat_drops: 

https://github.com/drone/drone/issues/1066